src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -39,7 +39,7 @@ namespace winrt
 namespace winrt::TerminalApp::implementation
 {
     TerminalPage::TerminalPage() :
-        _tabs{}
+        _tabs{ winrt::single_threaded_observable_vector<winrt::TerminalApp::Tab>() }
```

Comment:
this should be able to be properly constructed with just `_tabs{}`

Reply:
`_tabs{}` doesn't seem to initialize the vector properly, it sets it to `null` and eventually crashes when trying to `GetAt` on it. 😢